plugins/scaffolder-react/report-alpha.api.md

```diff
@@ -317,6 +317,7 @@ export type ScaffolderPageContextMenuProps = {
   onActionsClicked?: () => void;
   onTasksClicked?: () => void;
   onCreateClicked?: () => void;
+  onTemplateExtensionsClicked?: () => void;
```

Comment:
I'm thinking that we call this `templatingExtensions` instead.

`Template Extensions` is a bit ambiguous in naming I'm feeling.